src/lib/include/openE57/openE57.h

```diff
@@ -39,12 +39,17 @@
 #  include <stdlib.h>
 #endif
 
+#include <cinttypes>
 #include <iostream>
 #include <limits> // standard integers definition and numeric limits
 #include <memory>
 #include <string>
 #include <vector>
-#include <cinttypes>
+
+#if defined(WIN32)
```

Comment:
aren't these headers only for MinGW required? Under Visual Studio they aren't necessary, as the compilation works out of the box. Here should be

`#if defined(WIN32) && defined(__GNUC__)`